browser/ui/BUILD.gn

```diff
@@ -55,8 +55,8 @@ source_set("ui") {
   # It doesn't make sense to view accelerators on iOS & Android.
   if (!is_android && !is_ios) {
     sources += [
-      "webui/webcompat_reporter_ui.cc",
-      "webui/webcompat_reporter_ui.h",
+      "webui/webcompat_reporter/webcompat_reporter_ui.cc",
+      "webui/webcompat_reporter/webcompat_reporter_ui.h",
     ]
   }
```

Comment:
nit but it might be worth updating the comment here, looks like it used to apply to something else